tests/NPSP.py

```diff
@@ -11,12 +11,19 @@
 from SeleniumLibrary.errors import ElementNotFound
 from simple_salesforce import SalesforceMalformedRequest
 from simple_salesforce import SalesforceResourceNotFound
-from locator_w19 import npsp_lex_locators
 from selenium.webdriver import ActionChains
 from cumulusci.robotframework.utils import selenium_retry
 import sys
 from email.mime import text
 
+from locators_44 import npsp_lex_locators as locators_44
+from locators_45 import npsp_lex_locators as locators_45
+locators_by_api_version = {
+    44.0: locators_44,  # Winter '19
```

Comment:
We'll just want to make sure that we prune this as new releases come out.  There's no reason to keep locators_44 after next weekend when 45 will be in the wild everywhere.